gno.land/pkg/gnoweb/gnoweb.go

```diff
@@ -507,3 +511,16 @@ func pathOf(diruri string) string {
 
 	panic(fmt.Sprintf("invalid dir-URI %q", diruri))
 }
+
+func filterQueryValues(values url.Values) url.Values {
+	filtered := url.Values{}
+	for k, v := range values {
+		// Filter out private gnoweb arguments
+		if strings.HasPrefix(k, "__") {
```

Comment:
Checking "__" to allow "_" prefix to users but otherwise checking "_" should be done

Reply:
This comment is deprecated because `gnoweb` specific arguments are specified using the `$` symbol now